src/components/TeamMemberTasks/components/TaskCompletedModal.jsx

```diff
@@ -1,31 +1,74 @@
-import React from 'react';
 import { Button, Modal, ModalHeader, ModalBody, ModalFooter } from 'reactstrap';
+import React from 'react';
+import { useEffect } from 'react';
+import { ENDPOINTS } from 'utils/URL';
+import axios from 'axios';
 
 /**
  * Modal popup to delete the user profile
  */
-const DuplicateNamePopup = React.memo(props => {
+const TaskCompletedModal = React.memo(props => {
   const closePopup = e => {
-    props.onClose();
+    props.popupClose();
+  };
+
+  const loadUserTasks = async id => {
+    const userId = id;
```

Comment:
Please, delete this userId and use just id to make the code cleaner. Thank you very much!

Reply:
@rauleffting deleted. Please review again.

Reply:
Hi @LeeSingChanYau.  Is still there...

const loadUserTasks = async id => {
---->const userId = id; <----
    axios

Reply:
Hi @LeeSingChanYau,
The line @rauleffting mentioned is still in the code. Will this be changed or kept?